crates/piwis-val/src/lib.rs

```diff
@@ -81,7 +81,7 @@ pub struct VehicleIdentity {
 }
 
 #[derive(Deserialize, Serialize, Debug)]
-#[serde(deny_unknown_fields, rename_all = "SCREAMING_SNAKE_CASE")]
```

Comment:
i'd prefer to leave this one in and update the types wherever there needs to be changes. i can help with that if you want to send me a copy or if you are able to figure out how i can generate one

Reply:
sure thing.